receiver/googlecloudpubsubpushreceiver/receiver.go

```diff
@@ -271,6 +301,13 @@ func handlePubSubPushRequest[T any](
 		unmarshalData = request.Message.Data
 	}
 
+	if tb != nil {
+		incomingSize := len(unmarshalData)
+		bucketName := request.Message.Attributes[bucketIDKey] // it will be empty if coming from Pub/Sub directly
+		tb.GooglepubsubpushInputUncompressedLogSize.Record(ctx, float64(incomingSize), metric.WithAttributeSet(
+			attribute.NewSet(attribute.String(bucketNameAttr, bucketName))),
+		)
```

Comment:
If we use streaming in the future instead of loading the whole file, this metric might be gone